pulp_deb/app/tasks/copy.py

```diff
@@ -0,0 +1,119 @@
+from django.db import transaction
+from django.db.models import Q
+
+from pulpcore.plugin.models import Content, RepositoryVersion
+
+from pulp_deb.app.models import AptRepository, Package, ReleaseArchitecture, PackageReleaseComponent
+
+import logging
+from gettext import gettext as _
+
+log = logging.getLogger(__name__)
+
+
+def find_structured_publish_attributes(content, src_repo_version):
```

Comment:
```suggestion
def find_structured_publish_content(content, src_repo_version):
```
Final naming nitpick.